WikiWatershed/model-my-watershed
# Cinergi Result List View

## Overview
[Template here](https://docs.google.com/document/d/1NzDzEQ9a0cW3aEDRtrS9tHxazYsq6ScaYsnKOf7S3mw/edit)

Adds and displays new CINERGI fields in a CINERGI-specific search result. The displayed date is now the `timeperiod_nst` field which is what the date filters were already searching on.

Connects #2392 
### Demo

![screen shot 2017-10-24 at 3 33 32 pm](https://user-images.githubusercontent.com/7633670/31964136-1728c744-b8d1-11e7-9c77-ca13af496a8f.png)

### Notes

#### Many dates are garbage: 1/1/1900
We might want to filter these values out.

<details>

<summary>pic </summary>

![screen shot 2017-10-24 at 3 34 06 pm](https://user-images.githubusercontent.com/7633670/31964199-47e0869c-b8d1-11e7-8d17-ff9da709b404.png)

</details>

#### Some dates have same start and end
We might want to display these as a single date instead of range.

<details>

<summary>pic </summary>

![screen shot 2017-10-24 at 3 34 54 pm](https://user-images.githubusercontent.com/7633670/31964209-51023a22-b8d1-11e7-968d-9f915a07bc94.png)

</details>

#### Some raw descriptions contain html

I somewhat addressed this in 73f398f. Was surprised nunjunks didn't escape the html characters for us.

<details>

<summary>pics</summary>

rendering weird
![screen shot 2017-10-23 at 5 13 27 pm](https://user-images.githubusercontent.com/7633670/31964428-f6b2be42-b8d1-11e7-99d2-00599b0f3fa5.png)

dom
![screen shot 2017-10-23 at 5 14 00 pm](https://user-images.githubusercontent.com/7633670/31964429-f6bd576c-b8d1-11e7-85c8-2c54bb424dc6.png)
</details>

#### A lot of sources are `ScienceBase_processed`, which is ugly

<details>
<summary>pic</summary>

![screen shot 2017-10-24 at 3 53 13 pm](https://user-images.githubusercontent.com/7633670/31965072-21972f7e-b8d4-11e7-8c05-9f30bfae451b.png)

</details>

#### Description: 1-line vs 2-lines
It was a little unclear in the template if the description should be shortened to one line. We can drop 
faadb1c if we'd prefer to keep it the length it was.

## Testing Instructions

 * Pull and `bundle`
 * Search somewhere with CINERGI results and visually check the results
 * Confirm you can use the date filters in all sorts of combinations and that the dates in the results conform to the filters
